### PR TITLE
fuzz: fix coverity warnings

### PR DESCRIPTION
**Pull request by catenacyber**
cf https://github.com/openssl/openssl/pull/20128#issuecomment-1423466180

### CODE REVIEW

File: ssl/record/methods/tls_common.c [hunk 1]
```diff
@@ -865,7 +865,8 @@ int tls_get_more_records(OSSL_RECORD_LAYER *rl)
             if (thisrr->length > SSL3_RT_MAX_COMPRESSED_LENGTH + mac_size)
                 enc_err = 0;
 #ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
-            if (enc_err == 0 && mac_size > 0 && (md[0] ^ thismb->mac[0]) != 0xFF) {
+            if (enc_err == 0 && mac_size > 0 && thismb &&
```
Comment (paulidale) [inline ssl/record/methods/tls_common.c:868]: `thismb != NULL` according to our coding style.  Likewise `thismb->mac != NULL` on the next line.

**Review by paulidale — APPROVED** [review]
With the two little style nits addressed.

Thanks!

File: ssl/record/methods/tls_common.c (hunk 1 above)
Comment (catenacyber) [reply ssl/record/methods/tls_common.c]: ok, fixed up

**Review by paulidale — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by tmshort** [comment]
Merged to master branch. Thank you for your contribution!

**Pull request closed without merging**